test/functional/admin/legacy_responses_controller_test.rb

```diff
@@ -1,116 +0,0 @@
-require "test_helper"
```

Comment:
So we should leave this file as is so we continue to test the Bootstrap implementation until we release the page in  the design system. At that point we can remove this file.

Reply:
Oh yeah, that makes sense!! I've restored that file.